models/bamf_nnunet_mr_liver/config/default.yml

```diff
@@ -0,0 +1,42 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for Bamf NNUnet MR Liver segmentation (dicom to dicom)
+
+execute:
+- DicomImporter
+- NiftiConverter
+- NNUnetRunner
+- BamfProcessorRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: 'mr'
+
+  NiftiConverter:
+    in_datas: dicom:mod=mr
+    engine: dcm2niix
+    
+  NNUnetRunner:
+    in_data: nifti:mod=mr
+    nnunet_task: 'Task822_Liver_AMD'
+    nnunet_model: '3d_fullres'
+    roi: LIVER
+
+  BamfProcessorRunner:
```

Comment:
Line can be removed if no configuration is specified (will still be executed, as it's listed in the `execute` chain).